build.sh

```diff
@@ -17,6 +17,7 @@ debug_opts='--enable-debug'
 user_opts=''
 build_cflags='-O0 -g'
 job_count='4'
+make_tool='make'
```

Comment:
nit: I still think `gnu_make` or just `make` would be a better choice. But current name is okay.